apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -51,15 +51,15 @@ export const getItineraryType = (routes: ?Array<Array<string>>) => {
 };
 
 export const getItineraryDeparture = (sectors: ?Array<Sector>): ?RouteStop => {
-  return head(head(sectors ?? [])?.segments ?? [])?.departure;
+  return head(head(sectors ?? []).segments ?? []).departure;
 };
 
 export const getItineraryArrival = (
   tripType: ?TripTypes,
   sectors: ?Array<Sector>,
 ): ?RouteStop => {
   return tripType === TRIP_TYPES.RETURN
-    ? last(head(sectors ?? [])?.segments ?? [])?.arrival
+    ? last(head(sectors ?? []).segments ?? [])?.arrival
```

Comment:
Ad Albert mentioned. This part looks weird. It should return undefined according to docs https://ramdajs.com/docs/#head , so there is probably some mistake in new `ramda` type definition.

Reply:
Yep, it is the wrong definition in new the ramda type

Reply:
The issue raised https://github.com/flow-typed/flow-typed/issues/3355

Reply:
👍 Thanks. Since we are using our local copy we can probably fix it there directly if we need/want this PR merged.

Reply:
OK I will handle that 😸